packages/clarity-js/src/interaction/encode.ts

```diff
@@ -37,6 +37,7 @@ export default async function (type: Event, ts: number = null): Promise<void> {
                     tokens.push(pTarget.id);
                     tokens.push(entry.data.x);
                     tokens.push(entry.data.y);
+                    if (entry.data.id !== undefined) { tokens.push(entry.data.id); }
```

Comment:
When would this be undefined?

Reply:
When it's a mouse (click) event, not touch.